rmgpy/__init__.py

```diff
@@ -116,8 +115,7 @@ def load(self, path=None):
             elif os.path.exists(os.path.join(working_dir, 'rmgrc')):
                 self.filename = os.path.join(working_dir, 'rmgrc')
             else:
-                return  # fail silently, instead of raising the following error:
-                raise SettingsError('Could not find an RMG settings file to load!')
+                return  # fail silently, instead of raising an error
```

Comment:
Do we want to fail silently? If someone were to delete the `rmgrc` file, would anything work?

Reply:
So the comment associated with the last change is the following:

```
Quietly give up if can't find a settings file.  

The default one is empty anyway ;-) We may find a 
smarter way to do this, and indeed a better place to 
store settings, within anaconda apps.
```

The comment is correct--there is nothing but comments in the current rmgrc file, so by default we do not use this file to assign the correct path to the database directory and test_data directory. I have also confirmed that all of the unit tests pass if we delete this file.

How should we proceed? I think that we should either allow this to fail silently, or make it such that the user would have to specify the path to this file, as it is a rather niche feature, and we wouldn't want a settings file like this to be loaded unintentionally.

Reply:
I see. I think we should fail silently for now and rethink exactly what we want out of the config file in the future.